observability/akka-http-metrics/src/main/scala/com/daml/metrics/http/DamlWebSocketMetrics.scala

```diff
@@ -15,16 +15,24 @@ class DamlWebSocketMetrics(metricsFactory: Factory, component: String) extends W
   )
 
   override val messagesReceivedTotal: Meter =
-    metricsFactory.meter(httpMetricsPrefix :+ "websocket" :+ "messages" :+ "received")
+    metricsFactory.meter(
+      httpMetricsPrefix :+ "websocket" :+ "messages" :+ "received",
+      "Total number of received WebSocket messages",
```

Comment:
I would suggest exposing help text as full sentences (see example [here](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-format-example)):
```suggestion
      "Total number of received WebSocket messages.",
```